packages/devtools_app/lib/src/screens/inspector/inspector_tree_controller.dart

```diff
@@ -918,39 +933,32 @@ class _InspectorTreeState extends State<InspectorTree>
   /// Placeholder viewport height to use if we don't yet know the real
   /// viewport height.
   static const _placeholderViewportHeight = 1000.0;
+  static const _placeholderViewportWidth = 1000.0;
```

Comment:
instead of creating a second var, just have one called `_placeholderViewportSize`